### ISSUE TITLE
Outlook One: Message is saved to drafts without any prompt when user moves away during add-in processing

### ISSUE THREAD

**Issue by carlosb1504** [OfficeDev/office-js#3879]
# Provide required information needed to triage your issue

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Outlook One (New Outlook) on PC
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Operating System: Windows 10
 
## Expected behavior
Dialog should be shown inline with behaviour on Windows / OWA:
![image](https://github.com/OfficeDev/office-js/assets/47693383/0f562487-d168-4ff6-9a73-ccf33377a798)

## Current behavior

Message is saved to drafts without any warning to the user, creating confusion and inconsistency with the same add-in on other platforms

## Steps to reproduce

1. Compose an inline reply or forward of a message with any add-in which processes OnSend
2. Click away from the message whilst the ItemSend routine is running
3. Note that send processing is cut off, the item isn't sent and is saved in Drafts. No warning to the user.

**Comment by exextoc** [comment]
Thanks for reporting this issue regarding On Send Add-in. This has been put in our backlog and we, unfortunately, do not have a timeline to share at this point.

Internal Tracking ID: 4142329

**Comment by akshayta23** [comment]
We have rolled out the changes to inform the user when they have navigated away. A dialog with the following message will be shown to the user - 

**Your email was not sent**
One or more add-ins could not complete processing your email after you navigated away from it. Your email has been moved to the Drafts folder. Resend your email from the Drafts folder and remain on it until the add-in completes processing.

This change should reach production soon ie before the end of Apr 2024. cc @shighosh-msft
